### DISCUSSION TITLE
Schema modification methods docs question

### DISCUSSION THREAD

**Discussion by bringel** [jeremyevans/sequel#2085] [General]
Extremely low priority, but in the schema modifications doc page should [this line](https://github.com/jeremyevans/sequel/blob/3a14e2550aeb2d415be10cfd2d897b913c970677/doc/schema_modification.rdoc?plain=1#L84) about the options hash say "second argument" if you're using the column type as a method and "third argument" if you're using the `column` method, instead of "third" and "fourth" respectively?

**Comment by jeremyevans** [comment]
You are correct, thanks for pointing that out.  Can you send a pull request to fix it?  If not, I can take care of it.

**Reply by bringel** [reply]
Yes I sure can, just wanted to make sure that I was thinking of the right thing.
